sklearn/cluster/bicluster.py

```diff
@@ -238,11 +238,10 @@ class SpectralCoclustering(BaseSpectral):
         (n_cpus + 1 + n_jobs) are used. Thus for n_jobs = -2, all CPUs but one
         are used.
 
-    random_state : int, RandomState instance or None, optional, default: None
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
+    random_state : int, RandomState instance or None (default)
+        Used for randomizing the singular value decomposition. Use an int
```

Comment:
Used for randomizing the singular value decomposition and the k-means initialization.